admin/templates/spam/email.html

```diff
@@ -0,0 +1,78 @@
+{% extends "base.html" %}
+{% load static %}
+{% block title %}
+    <title>Email</title>
+{% endblock title %}
+{# stylesheets #}
+{% block content %}
+    {{ block.super }}
+
+    <div class="container-fluid">
+        <div class="row">
+            <div class="btn-group col-sm-12">
+                <a href="{% url 'spam:spam' %}" class="btn btn-primary">
+                    Back to list
+                </a>
+                <a href="{% url 'spam:detail' comment.id %}"
```

Comment:
I'll add a ```simple_tag``` that I wrote a few years ago for google-search-style pagination to the JIRA story. Github won't let me attach py or html files.

google search style: < prev 7 8 9 10 11 **12** 13 14 15 16 17 next>